src/components/filters/AttributeFilter/AttributeDropdown.tsx

```diff
@@ -185,16 +168,19 @@ export class AttributeDropdownWrapped extends React.PureComponent<
             isListReady: false,
             listError: null,
             items: [],
-            selection: [],
-            isInverted: true,
+            selection: props.selection || [],
+            prevSelection: null,
+            isInverted: props.isInverted !== undefined ? props.isInverted : !props.selection.length,
+            wasInverted: null,
             searchString: "",
         };
 
         this.createMediaQuery(props.fullscreenOnMobile);
 
         this.onDropdownToggle = this.onDropdownToggle.bind(this);
         this.onApply = this.onApply.bind(this);
-        this.onClose = this.onClose.bind(this);
+        this.onCancel = this.onCancel.bind(this);
```

Comment:
FYI, you don't have to do this, if you use arrow function methods instead;

`private onCancel = () => {`

Arrow function methods are auto-bound

Reply:
Thx, updated.